app/src/pages/inside/common/infoPanel/viewTabs/viewTabs.jsx

```diff
@@ -110,7 +114,7 @@ export class ViewTabs extends Component {
       {
         id: UNIQUE_ERRORS_VIEW,
         title: formatMessage(messages[UNIQUE_ERRORS_VIEW]),
-        link: listViewLink,
+        link: {},
```

Comment:
do we really need a redirect to root? we can stay on the same page
you can use CSS ```pointer-events: none``` to prevent any event on the disabled button

Reply:
No redirection, we stay on the same page

Reply:
URL doesn't change after click on this tab?

Reply:
URL changes,  'pointer-events: none' doesn't match,  selectLocationState(state) refreshes the page